tests_and_analysis/test/euphonic_test/test_install.py

```diff
@@ -11,7 +12,9 @@ def test_license_is_installed(self):
             license_data = fp.readlines()
         assert 'GNU GENERAL PUBLIC LICENSE' in license_data[0]
 
+    @pytest.mark.phonopy_reader  # Needs pyyaml
     def test_citation_cff_is_installed(self):
+        import yaml
```

Comment:
Do you want a try-catch with import error here? Then you might be able to skip rather than fail?

Reply:
Good idea, done.